draft-js-mention-plugin/src/mentionSuggestionsStyles.css

```diff
@@ -16,3 +16,13 @@
   box-sizing: border-box;
   transform: scale(0);
 }
+
+.mentionSuggestionsHeading:global(.first) {
```

Comment:
that should work and then you also can get rid of the `.first` class: https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child